internify/src/assets/Populations/studentScript.py

```diff
@@ -0,0 +1,293 @@
+		'codingLanguages': reformatDictionary(languagesDict[i]),
+		'frameworks': reformatDictionary(frameworksDict[i]),
+		'workTools': reformatDictionary(workflowToolsDict[i]),
+		'concepts': genCSSkillsDict[i]['General CS Skills'].split(", "),
+		'location': location[random.randint(0, 24)],
+		'degree': {degree[random.randint(0, 19)] : degreeLocation[random.randint(0, 19)]}	
+		}
+		students.append(student)
+		i+=1
+	print(json.dumps(students))
+	print('finished assembling students!!')
+
+
+# Use this method to synthesize a new, realistic, group of students
+def manufactureStudents(numStudents):
+	i = 0
+	students = []
+	print('starting to manufacture students')
+	while i < numStudents:
+		student = {
+
+		'id': uuid.uuid4().hex,
+		'gpa': gpaRandomizer(sGPA[random.randint(0, 19)]),
+		'year': aYear[random.randint(0, 19)], 
+		'coop': coop[random.randint(0,9)],
+		'seeking': seeking[random.randint(0, 9)],
+		'citizenship': citizenship[random.randint(0, 9)],
+		'workExperience': workExperience[random.randint(0,19)],
+		'codingLanguages': {
+			'Java': javaExperience[random.randint(0,19)],
+			'Python': pythonExperience[random.randint(0,19)],
+			'C++': cppExperience[random.randint(0,19)],
+			'C': cExperience[random.randint(0,19)],
+			'C#': cppppExperience[random.randint(0,19)],
+			'JavaScript': javascriptExperience[random.randint(0,19)],
+			'TypeScript': typescriptExperience[random.randint(0,19)],
+			'HTML': htmlExperience[random.randint(0,19)],
+			'CSS': cssExperience[random.randint(0,19)],
+			'R': rExperience[random.randint(0,19)],
+			'MATLAB': matlabExperience[random.randint(0,19)],
+			'Mathematica': reformatElement(languagesDict[i]['Mathematica']),
+			'SQL': sqlExperience[random.randint(0,19)],
+			'Other': otherExperience[random.randint(0,19)]
+
+		},
+		'frameworks': {
+			'React': reactExperience[random.randint(0,19)],
+			'Node': nodeExperience[random.randint(0,19)],
+			'Angular': angularExperience[random.randint(0,19)],
+			'Bootstrap': bootstrapExperience[random.randint(0,19)],
+			'MongoDB': mongodbExperience[random.randint(0,19)],
+			'Ruby on Rails': rubyonrailsExperience[random.randint(0,19)],
+			'Google Cloud': googlecloudExperience[random.randint(0,19)],
+			'AWS': awsExperience[random.randint(0,19)],
+			'Docker': dockerExperience[random.randint(0,19)],
+			'Linux': linuxExperience[random.randint(0,19)],
+			'Unix': unixExperience[random.randint(0,19)],
+			'Other': otherExperience[random.randint(0,19)]
+
+		},
+		'workTools': {
+			'GitHub': githubExperience[random.randint(0,19)],
+			'Azure': azureExperience[random.randint(0,19)],
+			'Jira': jiraExperience[random.randint(0,19)],
+			'Jupyter': jupyterExperience[random.randint(0,19)],
+			'Other': otherExperience[random.randint(0,19)]
+
+		},
+		'concepts': cssConceptsPopulator(),
+		'location': location[random.randint(0, 24)],
+		'degree': {degree[random.randint(0, 19)] : degreeLocation[random.randint(0, 19)]}	
```

Comment:
can you make the keys match what is currently in the JobPosting Model? easier for filtering through results and doing matching when keys match.